src/mockserver.js

```diff
@@ -1,9 +1,12 @@
 var config = window.__karma__.config,
-	ui5config = (config && config.openui5) || {},
+	ui5config = (config && config.openui5.config) || {},
 	mockserverConfig = ui5config.mockserver || {};
 
 //workaround for crossroads/mockserver issue
 (function() {
+	if (typeof module === undefined) {
+		var module = '';
```

Comment:
Aggreed. So it should be:
```javascript
if (typeof module === undefined){
  var module = undefined;
}
```
If i don't add these lines I am still getting an Uncaught ReferenceError at line 9 because module is not defined.